src/main/java/com/jw/resources/OrderResource.java

```diff
@@ -24,9 +25,12 @@ public class OrderResource {
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     public OrdersResponse getAllOrders() {
+        //REVIEW-VINI: We can change these logs to debug, small explanation why - this is my opinion :) -: nowadays most of the companies/services has a good metrics and trace for our services
```

Comment:
I agree these logs would be unnecessary when app will be deployed somewhere and info about incoming requests will be added out of the box. On the other hand, I really like knowing when I start and finish processing the request, especially if the service is mainly for local usage, I'll keep them but adding the trace would be a nice feature here